app/controllers/users_controller.rb

```diff
@@ -1,7 +1,11 @@
 class UsersController < ApplicationController
   def show
     if current_user.token
-      @repos = Repo.generate(conn(current_user.token)).take(5)
+      raw_repos = GithubService.repos(current_user)
```

Comment:
Lets get rid of this raw repos

Reply:
yeah, good catch - I completely missed that. We should be able to get it when we merge - I suggest we do so locally?